source/exploring_catalog_and_datasets/04_getting_involved/using_api.rst

```diff
@@ -3,14 +3,9 @@ Using the API
 
 An API (Application Programming Interface) is an interface for programs to communicate and exchange data.
 
-.. admonition:: Note
-   :class: note
-
-   To know more about what APIs are and why and how they are used, you can read the article "`You don\'t need analogies to understand APIs <https://kitchen.opendatasoft.com/you-dont-need-analogies-to-understand-apis-14da4513f970>`_" written by one of our Opendatasoft developers.
-
 Opendatasoft provides access to 6 APIs:
 
-- `ODS Search API v1 <https://help.opendatasoft.com/apis/ods-search-v1/>`_ and `ODS Search API v2 <https://help.opendatasoft.com/apis/ods-search-v2/>`_: Opendatasoft's own APIs, used internally by the platform and the `ODS widgets <https://help.opendatasoft.com/widgets/>`_
+- `ODS Explore API V2 <https://help.opendatasoft.com/apis/ods-explore-v2/>`_: Opendatasoft's own APIs, used by the platform.
```

Comment:
I think we should also highlight that it's our **main** public API here. Something like
```suggestion
- `ODS Explore API V2 <https://help.opendatasoft.com/apis/ods-explore-v2/>`_: main Opendatasoft explore API, used to explore catalogs and datasets with a custom SQL-like query language: ODSQL. Available publicly and also used internally.
```

Reply:
+1 but IMO we don't need to say it is used internally in the doc.